x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_table/index.tsx

```diff
@@ -448,7 +448,7 @@ const DetectionEngineAlertsTableComponent: FC<Omit<DetectionEngineAlertTableProp
               columns={finalColumns}
               browserFields={finalBrowserFields}
               onUpdate={onUpdate}
-              onLoaded={onLoad}
+              onLoaded={({ alerts }) => onLoad(alerts)}
```

Comment:
Could you extract this into a useCallback to prevent unnecessary rerenders?

```typescript
const onLoaded = useCallback(({ alerts }: AlertsTableOnLoadedProps) => onLoad(alerts), [onLoad]);
```